CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# v6.2.0 (2025-04-01)
+
+## OS Changes
+* Update readline from 8.2 to 8.2.13 ([#446])
+* Update glibc from 2.40 to 2.41 ([#437])
+* Remove bottlerocket-variant crate ([#435])
```

Comment:
Move the `bottlerocket-variant` note to a `## Build Changes` section

Reply:
Done.